### ISSUE TITLE
[BUG/CONFLICT] White (anti-aliasing-like) artifacts with Rubidium

### ISSUE THREAD

**Issue by moninformateur** [Sweenus/SimplySwords#18]
### Steps to reproduce (versions used noted below)
1. Enable Rubidium.
2. Enable Simply Sword.
3. Enter a new world.
4. Place multiple oak leaves to easily visualize the problem (see screenshots below).
5. White "anti-aliasing" artifacts appears.
6. Disable either Rubidium or Simply Sword and the visual glitches does not appear anymore. 
7. Downgrade Simply Sword to any version < 1.33 to fix the visual glitch.

### Other remarks
This bug is not present in any 1.19.2 versions of Simply Sword prior to 1.33

This bug has likely been introduced in version 1.33 of Simply Sword, as prior versions do not exhibit the visual glitches.

### Versions
Minecraft: 1.19.2
Forge: 43.2.1
Architectury: 6.4.62
Rubidium: 0.6.2
Simply Sword: 1.34-1.19.x
Nvidia drivers: 527.56

### Example

While the artifacts are very noticeable in game when any motion is involved, they do not survive any video compression. Here are screenshots demonstrating the glitches:

![2022-12-27 000134](https://user-images.githubusercontent.com/79221061/209613878-b6f88af1-4f16-420f-89a2-fce66e2d8975.png)

![2022-12-27 000332](https://user-images.githubusercontent.com/79221061/209613995-5f26b76a-fb5d-48a1-9ec3-aca0959a07b1.png)

**Comment by Sweenus** [comment]
Thanks for the detailed report! It looks like it's probably related to the mipmap bug introduced in version 1.33.
The mipmap bug will be fixed in version 1.35 (due in the next few days) - I may ask you to check if the issue persists once I push the update live.

**Comment by Sweenus** [comment]
Version 1.35 is on CurseForge now - pushed it a bit earlier than anticipated.
When you have time, can you please let me know if this has resolved the issue on your end?

**Comment by moninformateur** [comment]
Greetings and thank you for taking the time. Truly love your mod.

I've made some tests again with 1.35 and 1.36 following the same procedure as before, and the visual glitches seems to still persist. I'm starting to believe this may be caused by an issue coming from Rubidium, even if it isn't present in Simply Swords versions < 1.33. Truly weird.

I've opened [an issue on their end,](https://github.com/Asek3/Rubidium/issues/380) feel free to close this one if you also believe it to be on Rubidium's part.
